tck/features/clauses/create/Create1.feature

```diff
@@ -0,0 +1,137 @@
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Create1 - Creating nodes
+
+  Scenario: [1] Creating a single node
+    Given any graph
+    When executing query:
+      """
+      CREATE ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes | 1 |
+
+  Scenario: [2] Creating two nodes
+    Given any graph
+    When executing query:
+      """
+      CREATE (), ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes | 2 |
+
+  Scenario: [3] Creating a single node with a label
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (:Label)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 1 |
+
+  Scenario: [4] Creating a single node with multiple labels
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (:A:B:C:D)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 4 |
+
+  Scenario: [5] Creating a single node with a property
+    Given any graph
+    When executing query:
+      """
+      CREATE ({created: true})
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +properties | 1 |
+
+  Scenario: [6] Creating a single node with a property and returning it
+    Given any graph
+    When executing query:
+      """
+      CREATE (n {name: 'foo'})
+      RETURN n.name AS p
+      """
+    Then the result should be, in any order:
+      | p     |
+      | 'foo' |
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +properties | 1 |
+
+  Scenario: [7] Creating a single node with two properties
+    Given any graph
+    When executing query:
+      """
+      CREATE (n {id: 12, name: 'foo'})
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +properties | 2 |
+
+  Scenario: [8] Creating a single node with two properties and returning them
+    Given any graph
+    When executing query:
+      """
+      CREATE (n {id: 12, name: 'foo'})
+      RETURN n.id AS id, n.name AS p
+      """
+    Then the result should be, in any order:
+      | id | p     |
+      | 12 | 'foo' |
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +properties | 2 |
+
+  Scenario: [9] Creating a single node with null properties should not return those properties
```

Comment:
Somewhat similar as for 86 and co. However, here I suggest definitely doing the control query thing... i.e create a node with a null prop (only, and then checking that the side effects are correct), and then a control query returning it.

Reply:
see reply to comment for line 86; the problem here though is, that the result does not test the null. To complete the scenario also the name needs to be projected.